src/Datasource/RepositoryInterface.php

```diff
@@ -161,6 +161,17 @@ public function save(EntityInterface $entity, $options = []);
      */
     public function delete(EntityInterface $entity, $options = []): bool;
 
+    /**
+     * This creates a new entity object.
+     *
+     * Careful: This does not trigger any field validation.
+     * This entity can be persisted without validation error as empty record.
+     * Always patch in required fields before saving.
+     *
+     * @return \Cake\Datasource\EntityInterface
+     */
+    public function createEntity(): EntityInterface;
```

Comment:
We use 'create' in a few other context to generate persisted records eg. `findOrCreate()` This re-uses that term but without the persistent aspect which could be confusing. An alternative name could be `makeEntity()`

Reply:
True - or newEmptyEntity(), blankEntity(), initializeEntity()?

Reply:
`newEmptyEntity()` is good name , but a bit long

Reply:
Method length is not too relevant with no arguments and IDEs to autocomplete IMO.